client/app/vps/cloud-database/vps-cloud-database.routing.js

```diff
@@ -1,8 +1,18 @@
+import { CLOUD_DATABASE_ERRORS, CLOUD_DATABASE_FEATURE } from './vps-cloud-database.constants';
+
 import component from './vps-cloud-database.component';
 
 export default /* @ngInject */($stateProvider) => {
   $stateProvider
     .state('iaas.vps.detail.cloud-database', {
+      resolve: {
+        isAuthorized: /* @ngInject */ (
+          $q,
+          capabilities,
+        ) => (capabilities.includes(CLOUD_DATABASE_FEATURE)
+          ? null
+          : $q.reject(CLOUD_DATABASE_ERRORS.notAuthorized)),
```

Comment:
Do you really need a `Promise` here ? You don't have a `Promise` in the other case

Reply:
Yes, Promise.rejection makes the transition fall in error. Without it, the route is still accessible